### ISSUE TITLE
Duplicate <style> in rendered html

### ISSUE THREAD

**Issue by tammai** [apexcharts/vue-apexcharts#110]
I don't know why I have 2 style tags in head. When built, they didn't extracted to vendor-*.css too. Does anyway to load CSS separately or exclude it?

**Comment by junedchhipa** [comment]
No, currently CSS is injected directly from the .js file to avoid loading it separately.
Did you face any issues if it is done this way?
